python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -267,6 +267,39 @@ def test_actor_method_bind_same_constant(ray_start_regular):
     compiled_dag.teardown()
 
 
+def test_actor_method_bind_same_arg(ray_start_regular):
```

Comment:
Also add tests for:
- same task reads one input twice
- check that serialization context is empty at the end of a DAG execution (or ideally once the last task to read an input has started)

Reply:
Done.